scripts/build/builders/bouffalolab.py

```diff
@@ -136,20 +136,23 @@ def __init__(self,
         if (enable_ethernet or enable_wifi) and enable_thread:
             raise Exception('Currently, Thread cannot be enabled with Wi-Fi or Ethernet')
 
+        if enable_thread:
+            chip_mdns = "platform"
+        elif enable_ethernet or enable_wifi:
+            chip_mdns = "minimal"
+
         # hardware connectivity support check
         if bouffalo_chip == "bl602":
             if enable_ethernet or enable_thread:
-                raise Exception('SoC %s doesn\'t support connectivity Ethernet/Thread.' % bouffalo_chip)
-
+                raise Exception('SoC {} doesn\'t support connectivity Ethernet/Thread.'.format(bouffalo_chip))
         elif bouffalo_chip == "bl702":
             self.argsOpt.append('module_type=\"{}\"'.format(module_type))
```

Comment:
here the escaping of " does not seem needed...

Reply:
Updated